test/types/records/bad-change-type/range-init2.good

```diff
@@ -1,3 +0,0 @@
-range-init2.chpl:1: In function 'test':
-range-init2.chpl:6: error: invalid copy-initialization
-range-init2.chpl:6: error: cannot initialize a non-stridable range from a stridable range
```

Comment:
This and the above bad-change-type directory tests should continue to issue an error - that means they should use the `var err: range(stridable=false)`.

Reply:
Done in https://github.com/chapel-lang/chapel/pull/18268/commits/e8ebaf549c830eab48ca67356d95c78ced3fa889, thanks!